### PR TITLE
docs: 'How to deploy Grafana' manual

### PR DESCRIPTION
**Pull request by RonitKissis**
Added a manual to enable easy deployment of the grafana stack locally for easier development and testing.

| Question                | Answer                                                                          |
| ---------------- | -------------------------------------------------------------------------- |
| Bug fix         | ✖                                                                        |
| New feature     | ✖                                                                        |
| Breaking change | ✖                                                                        |
| Deprecations    | ✖                                                                        |
| Documentation   | ✔                                                                        |
| Tests added     | ✖                                                                        |
| Chore            | ✖                                                                       |

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Telemetry
 ## Motive
 This package goal is to make the experience of configuring and working with OpenTelemetry easier.
+
+### [Manual for easy local grafana deployment](./localManual.md)
```
Comment (CptSchnitz) [inline README.md:5]: I would put it in a ## title and not under `Motive`.

File: localManual.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+### 3. Enable Prometheus Collector from Your Local App
+
+- Open the cloned repository and navigate to `tempo/example/docker-compose/shared/prometheus.yaml`.
+- Add the following job according to the service you are running locally:
+
+    ```yaml
+    - job_name: 'app'
+      static_configs:
+        - targets:
+          - '172.17.0.1:8081'
```
Comment (CptSchnitz) [inline localManual.md:36]:
```suggestion
          - '172.17.0.1:8080'
```

File: localManual.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    ```
+
+    - Docker internal network IP that is static: `172.17.0.1`
+    - The port your service is running on: `8081`
```
Comment (CptSchnitz) [inline localManual.md:40]:
```suggestion
    - The port your service is running on: `8080`
```

**Review by CptSchnitz — CHANGES_REQUESTED** [review]

**Review by CptSchnitz — APPROVED** [review]

**Review by netanelC — CHANGES_REQUESTED** [review]
Put the file under docs and rename the file to "local-manual"

**Review by netanelC — APPROVED** [review]

**Pull request merged**